examples/keygen/keygen.c

```diff
@@ -178,6 +179,10 @@ int TPM2_Keygen_Example(void* userCtx, int argc, char *argv[])
             alg = TPM_ALG_SYMCIPHER;
             bAIK = 0;
         }
+        if (XSTRNCMP(argv[argc-1], "-keyedhash", 10) == 0) {
```

Comment:
XSTRNCMP unnecessary. argv guaranteed to be null-terminated strings. Use XSTRCMP.